dataflow/data/bigquery/batch-runs/historic/uploads_ip_by_day_base.json

```diff
@@ -1,14 +1,14 @@
 {
-  "projectId": "mlab-oti",
+  "projectId": "mlab-sandbox",
   "queryFile": "./data/bigquery/queries/base_uploads_ip_by_day.sql",
   "schemaFile": "./data/bigquery/schemas/base_uploads_ip.json",
-  "outputTable": "bocoup_prod.base_uploads_ip_by_day",
+  "outputTable": "data_viz.base_uploads_ip_by_day",
 
-  "lastDateFromTable": "[mlab-oti:plx.google:m_lab.ndt.all]",
+  "lastDateFromTable": "[plx.google:m_lab.ndt.all]",
```

Comment:
I was pretty sure this was an error that was never caught. The project name in this case is `plx.google`. It always appears the fixed way above in our queries. 

I just ran a test in BigQuery to verify and got this error:
```
Error: Invalid table name: mlab-oti:plx.google:m_lab:ndt.all
```